src/FactSystem/SettingsFact.cc

```diff
@@ -20,19 +20,19 @@ SettingsFact::SettingsFact(QObject* parent)
     QQmlEngine::setObjectOwnership(this, QQmlEngine::CppOwnership);
 }
 
-SettingsFact::SettingsFact(QString settingGroup, FactMetaData* metaData, QObject* parent)
-    : Fact(0, metaData->name(), metaData->type(), parent)
-    , _settingGroup(settingGroup)
-    , _visible(true)
+SettingsFact::SettingsFact(QString settingsGroup, FactMetaData* metaData, QObject* parent)
+    : Fact          (0, metaData->name(), metaData->type(), parent)
+    , _settingsGroup(settingsGroup)
+    , _visible      (true)
 {
     QSettings settings;
 
-    if (!_settingGroup.isEmpty()) {
-        settings.beginGroup(_settingGroup);
+    if (!_settingsGroup.isEmpty()) {
+        settings.beginGroup(_settingsGroup);
     }
 
     // Allow core plugin a chance to override the default value
-    _visible = qgcApp()->toolbox()->corePlugin()->adjustSettingMetaData(*metaData);
+    _visible = qgcApp()->toolbox()->corePlugin()->adjustSettingMetaData(settingsGroup, *metaData);
```

Comment:
Ok, so here is where it's actually called. Right?

Reply:
Ok, it's all working as it should. I've updated some random "custom" build and it's all working:

```
if (settingsGroup == AppSettings::settingsGroup && metaData.name() == AppSettings::virtualJoystickName) {
    metaData.setRawDefaultValue(false);
    return false;
}
```